src/main/java/address/MainApp.java

```diff
@@ -35,15 +41,29 @@ public void start(Stage primaryStage) {
     }
 
     protected void setupComponents() {
+
         config = getConfig();
         PreferencesManager.setAppTitle(config.appTitle);
-        AddressBookWrapper dataFromFile = StorageManager.getDataFromFile(
-                                                PreferencesManager.getInstance().getPersonFile());
+
+        final File initialFile = PreferencesManager.getInstance().getPersonFile();
+        AddressBookWrapper dataFromFile;
+        Exception fileLoadException = null;
+        try {
+             dataFromFile = StorageManager.loadDataFromSaveFile(initialFile);
+        } catch (JAXBException | FileContainsDuplicatesException e) {
+            fileLoadException = e;
+            dataFromFile = new AddressBookWrapper();
+        }
+
```

Comment:
Or we could just not inform the user of the duplicate problem when it occurs at first load. Then silently substitute sample data or empty data. But it seems wrong.